### ISSUE TITLE
Pretty awesome!Is it possible to consider the possibility of controlling the weight of the self-class control power individually when using it!相当棒！！是否可以考虑在使用的时候可以单独控制自类控制力的权重！

### ISSUE THREAD

**Issue by kenic123** [xinsir6/ControlNetPlus#2]
Pretty awesome!Is it possible to consider the possibility of controlling the weight of the self-class control power individually when using it!

**Comment by xinsir6** [comment]
In theory, it is. You can set the weight in Condition Transformer artificially. In default, the weight is set to 1.0 for each input condition and this is the same with training setting. One of our purpose to design this network architecture is to let people not set hyperparameter seriously but also can generate good result. The fusion of different conditions is naturally in training and can be adjusted by your application need.

**Comment by 3blackbar** [comment]
xinsir do You plan to train inpaint controlnet for sdxl ? Your models are the best , even better than sd1.5.

**Comment by xinsir6** [comment]
Yes, there is plan for it. If the repo over 500 stars, I will release a promax version with additonal 2 control, tile and inpating. Anyway, the model is being checked, before it releases, I will ensure it reaches the best performance.

**Comment by 3blackbar** [comment]
Thats amazing

**Comment by xinsir6** [comment]
In a union, also can be released in separate, but union will bright more convenience for people to use.

**Comment by kenic123** [comment]
> Pretty awesome!Is it possible to consider the possibility of controlling the weight of the self-class control power individually when using it!太棒了！使用时是否可以考虑单独控制自控电源重量的可能性！

I may not have made it clear that the UNION model is great, but the problem now is that you still need two control nodes and two preprocessors when you need to add more than two controls at the same time.If you can have A preprocessing node that can output different preprocessing results to the same controlnet node at the same time, and use a pull rod panel to control various weights, that is equivalent to A-a,B-b,C-c....The interface will be quite simple, and if it can be implemented, it will be a revolutionary means of control.
我可能没有表达清楚，这个UNION模型非常棒，但现在的问题是需要同时加两种以上控制的时候依然是需要两个control节点和两个预处理器。如果可以能有一个预处理节点，可以同时输出不同的预处理结果给到同一个controlnet节点，并且用一个拉杆面板控制各类权重，那就相当于A-a,B-b,C-c....,依次相连，这样界面将会相当简单，如果能够实现，那将是革命性的控制手段。
